src/avfoundation.cs

```diff
@@ -1078,14 +1318,17 @@ interface AVAudioOutputNode {
 
 	}	
 
-	[NoTV]
-	[NoWatch]
-	[iOS (8,0)][Mac (10,10)]
-	[BaseType (typeof (AVAudioIONode))]
-	[DisableDefaultCtor] // returned Handle is nil
-	// note: sample source (header) suggest it comes from AVAudioEngine properties
-	interface AVAudioInputNode : AVAudioMixing {
+	[Watch (4,0), TV (11,0), Mac (10,10), iOS (8,0)]
+	unsafe delegate IntPtr AVAudioIONodeInputBlock (uint frameCount);
 
+	[Watch (4,0), TV (11,0), Mac (10,10), iOS (8,0)]
+	[BaseType (typeof(AVAudioIONode))]
+	[DisableDefaultCtor]
+	interface AVAudioInputNode : AVAudioMixing
+	{
```

Comment:
diff is herd to read because the existing availability attributes were replaced
and because `{` was put on a different line